README.md

```diff
@@ -1 +1,123 @@
-# guru-discord-bot
\ No newline at end of file
+<h1 align="center">Guru Discord Bot</h1>
+<div align="center">
+    <strong>
+        Szerveren lévő jogosultságok/skillek kezelő botja a <a href="https://discord.gg/A5yC5Hhxrs">Step into Community</a> Discord szerveren.
+    </strong>
+</div>
+<br/>
+<div align="center">
+
+[![Discord](https://img.shields.io/discord/848942109191962665.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/A5yC5Hhxrs)
+
+
+</div>
+<br/>
+
+## Funkcionalitás
+
+- Csatlakozó felhasználók üdvözlése, ami után reakciók segítségével tudják magukhoz rendelni a szerepköröket
+- A bot által nyújtott parancsok/súgó lekérése (`/skills`)
+- Szerepkörök/skillek listázása a szerveren (`/list-skills`)
+- Szerepkörök felhaszálóhoz rendelése (`/i-know`)
+- Szerepkörök törlése/visszautasítása (`/i-dont-know`)
+
+## Jellemzők
```

Comment:
The listed stuff under Properties are more like TODOs in my opinion, might consider renaming this paragraph header.

Reply:
True, I used Attila's whoami readme.md as a reference but it is true that they are more likely to be TODOs than properties.

Reply:
I think this is fine. The previous section clearly lists what the bot is capable to do, while this section explains how it is done. These are not necessarily TODOs, I think, since the application can perfectly work as-is.